pandas/tests/dtypes/test_inference.py

```diff
@@ -59,7 +59,7 @@ def coerce(request):
     ({'a', 1},                  'set', 'set'),                  # noqa: E241
     (set(),                     'set', 'set-empty'),            # noqa: E241
     (frozenset({'a', 1}),       'set', 'frozenset'),            # noqa: E241
-    (frozenset([]),             'set', 'frozenset-empty'),      # noqa: E241
+    (frozenset(),               'set', 'frozenset-empty'),      # noqa: E241
```

Comment:
Is the noqa still needed?

Reply:
yes, this has to do with liniing things up.